### DISCUSSION TITLE
Using Logo

### DISCUSSION THREAD

**Discussion by tyama501** [jbruchon/elks#1609] [Show and tell]
Hello,

I am thinking to put a small advertisement of ELKS in a local event of my hometown.

Like
![elks_ad](https://github.com/jbruchon/elks/assets/61556504/46a73239-0fff-414d-b27b-e9cbe832907e)

I think using the ELKS Logo is ok,
but tell me if I am wrong.

Thank you.

**Comment by ghaerr** [comment]
Hello @tyama501,

That sounds fantastic! 

Hopefully your event will draw lots of interest to all the work you have done for the PC-9800 series computers. :)

Thank you!

**Reply by tyama501** [reply]
Thank you!
